Source/OxyPlot/Rendering/OxyPen.cs

```diff
@@ -108,7 +102,7 @@ public override int GetHashCode()
         {
             unchecked
             {
-                int result = this.Color.GetHashCode();
+                var result = this.Color.GetHashCode();
```

Comment:
Not a problem, but we should consider using `HashCode.Combine` for this sort of thing.